moveos/moveos-stdlib/moveos-stdlib/sources/object.move

```diff
@@ -28,10 +28,21 @@ module moveos_std::object {
         // The value must be the last field
         value: T,
     }
+  
+    /// An object ID
+    struct ObjectID has store, copy, drop {
+        // TODO should use u256 to replace address?
```

Comment:
In the current design and implementation, `address` is more readable than `u256`, so can this TODO be removed?